pkg/config/config.yaml

```diff
@@ -0,0 +1,6 @@
+apiServer: controller-runtime.sigs.k8s.io/v1alpha1
```

Comment:
is this testdata?  If so, it should be in pkg/config/testdata

Reply:
Done.